front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx

```diff
@@ -64,7 +85,19 @@ const ManchetteWithSpaceTimeChartWrapper = ({
         style={{ height: `${heightOfManchetteWithSpaceTimeChart}px` }}
         onScroll={handleScroll}
       >
-        <Manchette {...manchetteProps} />
+        <Manchette {...manchettePropsWithWaypointMenu}>
+          {waypointMenuData.menuPosition && (
+            <WaypointMenu
+              menuRef={waypointMenuData.menuRef}
+              items={waypointMenuData.menuItems}
+              style={{
+                width: '305px',
```

Comment:
Could we move this to a CSS file?

Reply:
This would require a div wrapper as the width it's not always the same when calling the menu. I'm not a huge fan of that. How would you do ?

Reply:
Wouldn't `.manchette .osrd-menu` work? If not, I'd suggest adding a `className` prop.

Reply:
Oh yes, you're right, done